src/main/kotlin/com/team4099/robot2023/config/constants/LedConstants.kt

```diff
@@ -35,26 +33,6 @@ class LedConstants {
     MOVEMENT(Blinkin.BlinkinLedMode.FIXED_STROBE_WHITE)
   }
 
```

Comment:
I deleted this cause according to phoenix6 migration guide candle support hasn't been added yet